### PR TITLE
Implement cart & address api

### PR DESCRIPTION
**Pull request by theMr17**
## Essential Checklist

- [ ] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR does not contain any unnecessary code changes from code editor.
- [x] The PR is made from a branch that's **not** called "master" and is up-to-date with "master".
- [x] The PR is **assigned** to the appropriate reviewers.

### CODE REVIEW

**Review by rahulc0dy — DISMISSED** [review]
SQL queries were not verified.

**Review by rahulc0dy — APPROVED** [review]
Check the results of search

File: src/database/queries/customer.query.js [hunk 1]
```diff
@@ -108,6 +108,73 @@ const updateCustomerPassword = async (email, passwordHash) => {
   }
 };
 
+const getCustomerAddressByAddressId = async (addressId) => {
+  const customer = await sql`
```
Comment (theMr17) [inline src/database/queries/customer.query.js:112]:
```suggestion
  const customerAddress = await sql`
```

File: src/controllers/customer.controller.js [hunk 2]
```diff
@@ -473,6 +479,216 @@ const updateCustomerImage = asyncHandler(async (req, res) => {
...
+            "No addresses found."
+          )
+        );
+    }
+
+    return res.status(200).json(
+      new ApiResponse({
+        address: isDefault && isDefault === true ? address[0] : address,
+      })
+    );
```
Comment (theMr17) [inline src/controllers/customer.controller.js:520]: message?

**Review by dattasneha — APPROVED** [review]

**Pull request merged**